src/index.ts

```diff
@@ -1,27 +1,34 @@
-import * as constants from './constants';
+import * as allConstants from './constants';
 import { focusInside } from './focusInside';
 import { focusIsHidden } from './focusIsHidden';
-import { getFocusMerge as focusMerge } from './focusMerge';
-import { getFocusabledIn, getFocusableIn } from './focusables';
+import { focusMerge } from './focusMerge';
+import { getFocusableIn } from './focusables';
 import { setFocus } from './setFocus';
 import { focusNextElement, focusPrevElement } from './sibling';
-import tabHook from './tabHook';
-import { getAllAffectedNodes } from './utils/all-affected';
 import { getActiveElement } from './utils/getActiveElement';
 
+/**
+ * magic symbols to control focus behavior from DOM
+ * see description of every particular one
+ */
+const constants = allConstants;
+
 export {
-  tabHook,
+  constants,
   focusInside,
   focusIsHidden,
+  setFocus,
```

Comment:
🤔 does not sounds right. If this is a default export from`focus-lock` package, then it should be `lockFocus`

Reply:
`react-focus-lock` was importing it as `moveFocusInside` and such name is better indication that this is not a "persistent" lock, but only a "move inside" action.